.github/workflows/ci-label.yml

```diff
@@ -0,0 +1,32 @@
+# Reference
+#   - https://github.com/actions/labeler
+#   - https://github.com/github/issue-labeler
+
+name: ci-label
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened]
+  issues:
+    types: [opened, edited, reopened]
+
+permissions:
+  issues: write
+  pull-requests: write
+  contents: read
```

Comment:
@HGWright From what I understand the default permissions for the GHA `GITHUB_TOKEN` is [permissive](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token).

So, just clarifying for my understanding... for `issues` and `pull-requests` you're not changing the permissions, but for `contents` you're restricting to `read` only, is that right?

Also, what's the reason for restricting the `contents` permissions? What's the implications?

Reply:
You are right @bjlittle, although the permissions were not set up by default on my fork. So I thought I needed them to make it work. I will remove them from the workflow.

As far as the `contents` is concerned to be honest it was with the others in the READ.ME for the issue-labeler action so I added it with the others. Off the top of my head the only thing I can think of is that the labeler would never need to write to the contents of a pr/issue, so I guess I can remove that too.

Reply:
I guess if less permissive permissions work, then just roll with that. Happy either way 👍